nonarch.imp

```diff
@@ -0,0 +1,20 @@
+[
+    #If the directory is not architecture specific, add this filter to avoid
+    #excess asm inclusions
+    { include: ["@.asm/ptrace.h.", private, "<linux/ptrace.h>", "public"] }, 
+    { include: ["@.asm-generic/bitops*.", private, "<linux/bitops.h>", "public"] },
```

Comment:
I don't understand the changes to filter.imp; How come you move some rules from there?

how come you move asm-generic/bitops -> linux/bitops here?